mlfinlab/labeling/__init__.py

```diff
@@ -4,5 +4,5 @@
 
 from mlfinlab.labeling.labeling import (add_vertical_barrier, apply_pt_sl_on_t1, barrier_touched, drop_labels,
                                         get_bins, get_events)
-from mlfinlab.labeling.trend_scanning import trend_scanning_labels
+from mlfinlab.labeling.trend_scanning import trend_scanning_labels, calculate_t_values
```

Comment:
Should `calculate_t_values` be a public method?

Reply:
not public anymore